src/EPR.Calculator.Frontend/appsettings.Development.json

```diff
@@ -16,5 +16,9 @@
   "DashboardCalculatorRun": {
     "RunParameterYear": "2024-25",
     "DashboardCalculatorRunApi": "http://localhost:5055/api/calculatorRuns"
+  },
+  "LapcapSettings": {
+    "ParameterYear": "2024-25",
+    "LapcapSettingsApi": "http://localhost:5055/api/lapcapData"
```

Comment:
We need to do this change
```
"LapcapSettings": {
    "ParameterYear": "2024-25",
    "LapcapPostEndPoint": "/api/lapcapData",
    "LapcapGetEndPoint": "/api/lapcapData/{parameterYear}"
}
```

Have another Global variable for Api environment which would have  "http://localhost:5055" 
And also code to concatenate the it.
Should be ok for this to be a separate ticket.

Reply:
@johnbenjamin-defra  before we forget can you please create a front-end ticket for doing this change.

Reply:
https://dev.azure.com/defragovuk/RWD-CPR-EPR4P-ADO/_workitems/edit/442022